.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,32 @@
+name: Run Link Checker
+on: [push]
+
+jobs:
+  initialize:
+    name: Clone Theme
+    runs-on: ubuntu-latest
+    steps:
+    - name: Initialize Theme
+      run: |
+        sudo apt-get update && sudo apt-get install -y git
+        git init
+        git submodule update --init --recursive --depth 1
+  build:
+    name: Build Website
+    runs-on: ubuntu-latest
+    steps:
+    - name: Build
+      env:
+        HUGO_VERSION: 0.55.6
+      run: |
+        sudo apt-get update && sudo apt-get install -y curl \
+                                                           git \
+                                                           openssh-client \
+                                                           rsync \
+                                                           npm && \
+                                                           npm install -G autoprefixer postcss-cli
+        curl -L https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_extended_${HUGO_VERSION}_Linux-64bit.tar.gz | tar -xz && \
+            sudo mv hugo /usr/local/bin/hugo
+        curl https://htmltest.wjdp.uk | sudo bash -s -- -b /usr/local/bin
+        hugo --config config.toml,linkcheck-config.toml --buildFuture
+        htmltest
```

Comment:
Ideally: run the link checker in a Kubernetes website container.

If that's not feasible, I'd like to copy the Hugo version from `config.toml`